pephub/routers/api/v1/project.py

```
@@ -1138,3 +1139,33 @@ def delete_full_history(
             status_code=400,
             detail="Could not delete history. Server error.",
         )
+
+
+@project.get(
+    "/standardize",
+    summary="Standardize PEP metadata column headers",
+)
+async def get_standardized_cols(
+    namespace: str, project: str, tag: Optional[str] = DEFAULT_TAG, schema: str = ""
+):
+    """
+    Standardize PEP metadata column headers using BEDmess.
+
+    Args:
+    - pep (str): PEP string to be standardized
+    - schema (str): Schema for AttrStandardizer
+
+    Returns:
+    - dict: Standardized results
+    """
+
+    if schema == "":
```

Can we raise an HTTP error here instead?
```python
from fastapi import HTTPException

if not schema: # or something.. if schema is None or schema == "":
    raise HTTPException(code=400, detail="Schema is required! Available schemas are ENCODE and Fairtracks")
```

---

added